### PR TITLE
Docs/update spacing

### PR DESCRIPTION
**Pull request by joasmi723**

### CODE REVIEW

File: packages/react-magma-docs/src/pages/design/card.mdx [hunk 1]
```diff
@@ -7,20 +7,17 @@ order: 1
...
+<LeadParagraph>
   Cards provide a flexible container with multiple variants and options.
-</Paragraph>
-
-<span class="horizontal-spacer"></span>
+</LeadParagraph>
 
 <figure>
-  <img src="../../images/cards/card-intro-example.png" alt=" " />
+  <img src="../../images/cards/card-intro-example.png" alt="" />
```
Comment (rcgillet) [inline packages/react-magma-docs/src/pages/design/card.mdx:20]: don't we want a space in the `alt` here for screen readers?

**Review by rcgillet — CHANGES_REQUESTED** [review]

File: packages/react-magma-docs/src/pages/design/card.mdx (hunk 1 above)
Comment (orion-cengage) [reply packages/react-magma-docs/src/pages/design/card.mdx:20]: If memory serves me correctly, I believe so.

Comment (joasmi723) [reply packages/react-magma-docs/src/pages/design/card.mdx:20]: Good catch!  Will change

File: packages/react-magma-docs/src/pages/api/checkboxes.mdx [hunk 2]
```diff
@@ -395,7 +394,7 @@ Placeholders are specific keywords surrounded by curly braces.
         status={status}
         labelText="Colors"
       />
-      <div style={{marginLeft: '30px'}}>
+      <div style={{marginLeft: '32px'}}>
```
Comment (avanslaars) [inline packages/react-magma-docs/src/pages/api/checkboxes.mdx:397]: Should this come out of the spacing object as well?

File: packages/react-magma-docs/src/pages/design/inputs.mdx [hunk 3]
```diff
@@ -5,30 +5,26 @@ order: 1
 ---
 
 import DocsHeading from '../../components/docs-heading';
+//import LeadParagraph from '../../components/lead-paragraph';
```
Comment (avanslaars) [inline packages/react-magma-docs/src/pages/design/inputs.mdx:8]: remove commented import

File: packages/react-magma-docs/src/pages/design/loading-indicators.mdx [hunk 4]
```diff
@@ -7,16 +7,15 @@ order: 1
 import { Link } from 'gatsby';
 import './design-guidelines-styles.css';
 import DocsHeading from '../../components/docs-heading';
-import { Paragraph } from 'react-magma-dom';
+import LeadParagraph from '../../components/lead-paragraph';
```
Comment (avanslaars) [inline packages/react-magma-docs/src/pages/design/loading-indicators.mdx:10]: I know we've been a bit inconsistent on this (the DocsHeading above is the same), but I'd like to move towards standardizing on named exports. Those are less likely to be renamed and that should help keep things consistent and lighten cognitive load when trying to follow the code.

File: packages/react-magma-docs/src/components/docs-heading/index.js [hunk 5]
```diff
@@ -43,25 +48,29 @@ const DocsHeading = ({ children, to, type }) => (
...
-            <PaletteIcon size="16" />
-            <ButtonSpan>View Design Guidelines</ButtonSpan>
-          </Link>
-        )}
-      </Hyperlink>
+      <ButtonContainer>
+        <Hyperlink color="secondary" styledAs="Button" to={to}>
+          {linkProps => (
+            <Link {...linkProps}>
+              <PaletteIcon size="16" />
```
Comment (avanslaars) [inline packages/react-magma-docs/src/components/docs-heading/index.js:55]: Should our icon sizes be coming out of the text scale?

File: packages/react-magma-docs/src/pages/design/loading-indicators.mdx (hunk 4 above)
Comment (joasmi723) [reply packages/react-magma-docs/src/pages/design/loading-indicators.mdx:10]: Sounds good -- yes, I was wondering about that myself.  I think I created the DocsHeading a long time ago, and I made this one match.   There are a few others in the docs as well.  I'd like to do this as a separate PR from this one, if you're okay with that.

File: packages/react-magma-docs/src/components/docs-heading/index.js (hunk 5 above)
Comment (joasmi723) [reply packages/react-magma-docs/src/components/docs-heading/index.js:55]: Good catch, yes. :)

File: packages/react-magma-docs/src/pages/api/checkboxes.mdx (hunk 2 above)
Comment (joasmi723) [reply packages/react-magma-docs/src/pages/api/checkboxes.mdx:397]: I've been less consistent about this in docs examples, but yes, I can definitely change this and will be more consistent going forward.

**Review by rcgillet — APPROVED** [review]

**Pull request merged**